Dockerfile

```diff
@@ -1,25 +1,38 @@
-FROM centos:7
+FROM phusion/baseimage
 
-RUN curl https://bintray.com/sbt/rpm/rpm > /etc/yum.repos.d/bintray-sbt-rpm.repo
-RUN yum -y install git sbt && yum clean all
+RUN add-apt-repository ppa:openjdk-r/ppa
```

Comment:
I have security concerns about this line (an untrusted ppa), but will defer to devops for their opinion